### PR TITLE
Add jarvis beta notice

### PR DESCRIPTION
**Pull request by alexcherman**

### CODE REVIEW

File: universal-login-wallet/src/ui/react/Home/WelcomeScreen.tsx [hunk 1]
```diff
@@ -1,18 +1,30 @@
...
 
 export const WelcomeScreen = () => {
   const {history} = useRouter();
+  const {sdk} = useServices();
+
+  const [notice, setNotice] = useState('');
+  useEffect(() => {
+    setBetaNotice(sdk);
+    setNotice(sdk.getNotice());
+  });
```
Comment (Marik-D) [inline universal-login-wallet/src/ui/react/Home/WelcomeScreen.tsx]:
```suggestion
  }, []);
```

File: universal-login-wallet/src/ui/react/Home/WelcomeScreen.tsx [hunk 2]
```diff
@@ -1,18 +1,30 @@
...
+import {Notice} from '@universal-login/react';
+import {setBetaNotice} from '@universal-login/sdk';
 
 export const WelcomeScreen = () => {
   const {history} = useRouter();
+  const {sdk} = useServices();
+
+  const [notice, setNotice] = useState('');
+  useEffect(() => {
+    setBetaNotice(sdk);
```
Comment (Marik-D) [inline universal-login-wallet/src/ui/react/Home/WelcomeScreen.tsx]: Move `setBetaNotice(sdk)` to `AppBootstraper` initialization code (right near `sdk.start()`).

And then this just becomes: `const notice = sdk.getNotice()`

Comment (marekkirejczyk) [reply universal-login-wallet/src/ui/react/Home/WelcomeScreen.tsx]: 👍

**Comment by alexcherman** [comment]
Screenshot 2019-10-17 at 09 52 29

**Pull request merged**